src/test/objectstore/store_test.cc

```diff
@@ -6984,6 +6984,34 @@ TEST_P(StoreTest, BluestoreRepairTest) {
   cerr << "Completing" << std::endl;
   bstore->mount();
 }
+TEST_P(StoreTest, BluestoreStatistics) {
+  if (string(GetParam()) != "bluestore")
+    return;
+
+  g_conf->set_val("rocksdb_perf", "true");
```

Comment:
Please use SetVal method to alter config parameters here and below.
This will automatically do default parameter restore on test case termination.